### PR TITLE
Fix GitHub action

### PR DESCRIPTION
**Pull request by jasonpaige**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/modules/ardoq/RequestProcessor.ts [hunk 1]
```diff
@@ -19,31 +24,66 @@ export class RequestProcessor {
   }
 
   public async processRequest(request: ArdoqRequest): Promise<Map<ArdoqComponentCreatedStatus, number>> {
```
Comment (linusnorton) [inline src/main/modules/ardoq/RequestProcessor.ts:26]: This method is pretty big now, could probably split some of it out

File: src/main/modules/ardoq/repositories/ArdoqBatchRespository.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { BatchRequest } from '../batch/BatchRequest';
+import { Component } from '../batch/Component';
+import { Reference } from '../batch/Reference';
+
+import { AxiosInstance } from 'axios';
+
+const { Logger } = require('@hmcts/nodejs-logging');
+
+export class ArdoqBatchRespository {
+  constructor(private httpClient: AxiosInstance, private logger = Logger.getLogger('ArdoqBatchRespository')) {}
```
Comment (linusnorton) [inline src/main/modules/ardoq/repositories/ArdoqBatchRespository.ts:12]: being pedantic but I always default to readonly unless I have a reason not to

File: src/main/modules/ardoq/repositories/ArdoqBatchRespository.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  private processBatchResponse(
+    components?: BatchResult,
+    references?: BatchResult
+  ): Map<ArdoqComponentCreatedStatus, number> {
+    const counts: Map<ArdoqComponentCreatedStatus, number> = new Map<ArdoqComponentCreatedStatus, number>([
+      [ArdoqComponentCreatedStatus.EXISTING, 0],
+      [ArdoqComponentCreatedStatus.CREATED, 0],
+    ]);
+
+    [...(components?.created ?? []), ...(references?.created ?? [])].forEach(u => {
```
Comment (linusnorton) [inline src/main/modules/ardoq/repositories/ArdoqBatchRespository.ts:43]: You could probably break this down to make it more readable. I'm also confused why it's happening twice with the true/false variable. Maybe do it in one iteration but with both calls?

**Review by linusnorton — APPROVED** [review]

File: src/main/modules/ardoq/RequestProcessor.ts (hunk 1 above)
Comment (jasonpaige) [reply src/main/modules/ardoq/RequestProcessor.ts]: I'll pick this up when I do the deletion feature as it's going to involve lots of work here anyway

File: src/main/modules/ardoq/ArdoqClient.ts [hunk 4]
```diff
@@ -109,7 +110,8 @@ export class ArdoqClient {
     return null;
   }
 
-  public async processBatchRequest(batchRequest: BatchRequest): Promise<Map<ArdoqComponentCreatedStatus, number>> {
+  public async processBatchRequest(batchRequest: BatchRequest): Promise<ArdoqStatusCounts> {
+    console.log('################################');
```
Comment (linusnorton) [inline src/main/modules/ardoq/ArdoqClient.ts:114]: do you want this in there?

**Pull request merged**